src/renderer/utils.rs

```diff
@@ -165,6 +152,20 @@ pub fn reflect_layout(context: &WebGl2RenderingContext, program: &GlProgram) ->
             .get_active_uniform_block_name(&program.program, uniform_index)
             .unwrap();
         if name == "Camera" {
+            let camera_descriptor = BindGroupDescriptor::new(
+                0,
+                vec![BindingDescriptor {
+                    name: "Camera".to_string(),
+                    index: 0,
+                    bind_type: BindType::Uniform {
+                        dynamic: false,
+                        property: UniformProperty::Struct(vec![UniformProperty::Mat4]),
+                    },
+                    shader_stage: BindingShaderStage::VERTEX | BindingShaderStage::FRAGMENT,
+                }],
+            );
+            used_indices.insert(camera_descriptor.index);
+            bind_groups.push(camera_descriptor);
             continue;
```

Comment:
While all Bevy shaders have a camera uniform, it's still possible to create a render node that would have a totally custom pipeline and shaders without a camera uniform.

I had to introduce this change to make my render pass work:
https://github.com/mvlabat/bevy_megaui/blob/73a0a0f497876fa1370d1c55fb5df2eb12c42a24/src/megaui_node.rs#L217